go/teams/service_helper.go

```diff
@@ -1157,6 +1157,23 @@ func removeMemberInviteOfType(ctx context.Context, g *libkb.GlobalContext, team
 	return libkb.NotFoundError{}
 }
 
+func removeKeybaseTypeInviteForUID(ctx context.Context, g *libkb.GlobalContext, team *Team, uid keybase1.UID) error {
+	g.Log.CDebugf(ctx, "looking for active or obsolete keybase-type invite in %s for %s", team.Name(), uid)
+
+	allInvites := team.GetActiveAndObsoleteInvites()
+	for _, invite := range allInvites {
+		if inviteUv, err := invite.KeybaseUserVersion(); err == nil {
+			if inviteUv.Uid.Equal(uid) {
+				g.Log.CDebugf(ctx, "found keybase-type invite %s for %s, removing it", invite.Id, invite.Name)
+				return removeInviteID(ctx, team, invite.Id)
```

Comment:
I don't think which one is the correct way. RemoveMember when user is cryptomember will also  remove only the first occurrence. I'm hesitant to changing both because we are leaning towards never allowing multiple occurrences of user in a team with a serverside check.

If it were to remove all invites for given uid, it shouldn't call `removeInviteID` for all of them but instead use something to remove all invites in one sig.

Do you think we should go with it? Only in this code path or in other member removal code as well?

Reply:
I don't know what the best thing is, but if there are multiple invites for a UID, what would this do if it only removes one of them?

It seems like if you are dropping the user version from part of the match, then you could remove the wrong one if you only remove one.

Reply:
oh, you're right.. the one for cryptomembers always finds the "latest" UV.

Reply:
@patrickxb fixed, can you take another look? Thanks!